packages/did-document/src/full/documentFull.ts

```diff
@@ -42,8 +43,8 @@ class DIDDocumentFull extends DIDDocumentLite implements IDIDDocumentFull {
    * ```
    * @return { boolean }
    */
-  async deactivate(): Promise<boolean> {
-    return this._operator.deactivate(this.did);
```

Comment:
how come this was working previously? async?

Reply:
return value from async function doesn't need to be prepended with `await`